src/locales/en.json

```diff
@@ -248,9 +247,9 @@
 "medium_size": "Medium",
 "menu": "Menu",
 "metadata_window_title": "Infobox",
-"meteoschweiz": "MeteoSwiss",
+"meteoschweiz": "MeteoSuisse",
 "meteoschweiz_service_link_href": "http://www.meteoswiss.admin.ch",
-"meteoschweiz_service_link_label": "MeteoSwiss",
+"meteoschweiz_service_link_label": "MeteoSuisse",
```

Comment:
I would prefer to change than in a separate PR, possibly linked to an issue. That's a bit out of the blue and I can't know if it's intended or not.

Reply:
OK, reverted MeteoSwiss changes